slides/03_workflow.qmd

```diff
@@ -1,10 +1,591 @@
 ---
-title: "A Package Engineering Workflow"
+title: "An R Package Engineering Workflow"
+subtitle: "BBS Course: Good Software Engineering Practice with R"
 author: "Friedrich"
+affiliation: "RPACT"
+institute: "RPACT"
+date: '2023-02-10'
+date-format: long
+tags: [R, Package]
+license: 'CC BY-SA'
+email-obfuscation: javascript
 format: 
-  revealjs
-description: description. Photo CC0 by Andrea Piacquadio on pexels.com.
+  revealjs:
+    theme: [default, custom.scss]
+    fontsize: 34px
+    toc: false
+    slide-number: true
+    progress: true
+    code-line-numbers: true
+    mouse-wheel: false
+    link-external-newwindow: true
+    footer: 'Good Software Engineering Practice with R - [Home](../index.html "GSWEP4R Workshop") | [Slides](listing.html "Overview of all slides") | [License](http://creativecommons.org/licenses/by-sa/4.0/ "License: CC BY-SA 4.0")'
```

Comment:
wow that is a lot of stuff there - I guess we could do this on the other slides too